dataland-frontend/src/components/resources/dataTable/conversion/YesNoDataPointValueGetterFactory.ts

```diff
@@ -0,0 +1,43 @@
+import { type Field } from "@/utils/GenericFrameworkTypes";
+import { type AvailableMLDTDisplayObjectTypes } from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import { type YesNoNa } from "@clients/backend";
+import { getDataPointGetterFactory } from "@/components/resources/dataTable/conversion/Utils";
+import { type GenericDataPoint } from "@/utils/DataPoint";
+
+const humanReadableYesNoMap: { [key in YesNoNa]: string } = {
```

Comment:
Why is this defined here? Also it is duplicated in the YesNoValueGetterFactory

Reply:
Adressed. There is now a central file defining humanized yes no and na values.